src/dbnode/persist/fs/files.go

```diff
@@ -1278,22 +1333,18 @@ type filesetFilesSelector struct {
 
 func filesetFiles(args filesetFilesSelector) (FileSetFilesSlice, error) {
 	var (
-		byTimeAsc []string
+		byTimeAsc sortedFilesetFiles
 		err       error
 	)
 	switch args.fileSetType {
 	case persist.FileSetFlushType:
 		switch args.contentType {
 		case persist.FileSetDataContentType:
 			dir := ShardDataDirPath(args.filePathPrefix, args.namespace, args.shard)
-			byTimeAsc, err = findFiles(dir, args.pattern, func(files []string) sort.Interface {
-				return dataFileSetFilesByTimeAndVolumeIndexAscending(files)
```

Comment:
`dataFileSetFilesByTimeAndVolumeIndexAscending` and `fileSetFilesByTimeAndVolumeIndexAscending` seems to be not used anymore after these changes, should be able to remove them, too.